tests/util/test_unit_conversion.py

```diff
@@ -295,7 +295,32 @@
         ),
     ],
     CarbonMonoxideConcentrationConverter: [
+        # PPB to other units
+        (
+            1,
+            CONCENTRATION_PARTS_PER_BILLION,
+            1000,
```

Comment:
`1 ppb == 0.001 ppm`
commit https://github.com/home-assistant/core/pull/159554/changes/4e9a074a76d489e6b71fdba3737f5b374664659c made the test wrong